.github/workflows/csv-coverage-pr-artifacts.yml

```diff
@@ -0,0 +1,85 @@
+name: Check framework coverage changes
+
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/csv-coverage-pr-comment.yml'
+      - '*/ql/src/**/*.ql'
+      - '*/ql/src/**/*.qll'
+      - 'misc/scripts/library-coverage/*.py'
+      # input data files
+      - '*/documentation/library-coverage/cwe-sink.csv'
+      - '*/documentation/library-coverage/frameworks.csv'
+    branches:
+      - main
+      - 'rc/*'
+
+jobs:
```

Comment:
To be extra careful you can add a `permissions` block to this job that restricts the scopes of the GITHUB_TOKEN to exactly what you need.

Reply:
I'm going to skip this as this workflow runs on `pull_request`, so it has max `read` rights on forked repos. BTW, do you know any good documentation on the permission scopes? https://docs.github.com/en/actions/reference/authentication-in-a-workflow#permissions-for-the-github_token
It doesn't seem to be documented.

Reply:
Sounds reasonable if we're only giving `read` rights to fork PRs.

https://docs.github.com/en/rest/reference/permissions-required-for-github-apps and https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#permissions have some more information.